src/screens/Settings/components/SyncSaves/gog.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { CircularProgress } from '@mui/material'
+import { fixGogSaveFolder, getGameInfo, ipcRenderer } from 'src/helpers'
+import ContextProvider from 'src/state/ContextProvider'
+import { GOGCloudSavesLocation, Path, SyncType } from 'src/types'
+import {
+  InfoBox,
+  SelectField,
+  TextInputWithIconField,
+  ToggleSwitch
+} from 'src/components/UI'
+import { Backspace, CreateNewFolder } from '@mui/icons-material'
+import { useTranslation } from 'react-i18next'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons'
+
+interface Props {
+  appName: string
+  gogSaves: GOGCloudSavesLocation[]
+  setGogSaves: (saves: GOGCloudSavesLocation[]) => void
+  autoSyncSaves: boolean
+  setAutoSyncSaves: (value: boolean) => void
+}
+
+export default function GOGSyncSaves({
+  appName,
+  gogSaves,
+  setGogSaves,
+  autoSyncSaves,
+  setAutoSyncSaves
+}: Props) {
+  const [isLoading, setIsLoading] = useState(true)
+  const [isSyncing, setIsSyncing] = useState(false)
+  const [syncType, setSyncType] = useState('--skip-upload')
+
+  const { t } = useTranslation()
+
+  const { platform } = useContext(ContextProvider)
+  const isWin = platform === 'win32'
+  const isMac = ['osx', 'Mac']
```

Comment:
This name could be better, with `isMac` you'd think it's a boolean
Maybe `macPlatforms`?